### ISSUE TITLE
Why data-evaluate-expression for variables [question]

### ISSUE THREAD

**Issue by royalstream** [WebFreak001/code-debug#58]
`data-evaluate-expression` is used to inspect the value of variables during debugging (as your documentation clearly says).

For some reason **gdb** still uses my pretty printers but **lldb-mi** ignores them.

This is not a bug report since I get the same behavior in the command line:  `data-evaluate-expression`  ignores my _type summary_ variable formatters but if I use `frame variable` or even `print` they come out nice.  Why is  `data-evaluate-expression` being used across the board? Is this intentional?

**Comment by WebFreak001** [comment]
`data-evaluate-expression` is actually the `print` equivalent in GDB/MI. Are you sure `print` and `frame variable` actually work in lldb? They don't for me, which would explain why variables don't work in LLDB

**Comment by royalstream** [comment]
Yes they work for me, but I don't know if it's something specific to **/Applications/Xcode.app/Contents/Developer/usr/bin/lldb-mi** (I'm using a mac).
